tapcfg/server.go

```diff
@@ -94,7 +107,7 @@ func genServerConfig(cfg *Config, cfgLogger btclog.Logger,
 	tapdbAddrBook := tapdb.NewTapAddressBook(
 		addrBookDB, &tapChainParams, defaultClock,
 	)
-	assetStore := tapdb.NewAssetStore(assetDB, defaultClock)
+	assetStore := tapdb.NewAssetStore(assetDB, metaDB, defaultClock, dbType)
```

Comment:
Seems like we need to add these new arguments in some unit tests as well, see CI failures.

Reply:
yeap the MetaDB was not instantiated there, added :+1: